benchmarks/asv_by_release.bash

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+RELEASE_LIST=$(git tag -l "release-202[43]*" | sort -r)
```

Comment:
Is this limited to 2023-2024 releases?

Reply:
No, it is open to anything. Basically, it is a pattern that we can perform.

For all tags in the years (2020-2029), it could be `"release-202[0-9]*"`
For all tags in the last month, it could be `"release-2024.03.*"`

---

The documentation is here: https://git-scm.com/book/en/v2/Git-Basics-Tagging#_listing_your_tags.

> You can also search for tags that match a particular pattern. The Git source repo, for instance, contains more than 500 tags. If you’re interested only in looking at the 1.8.5 series, you can run this:
> 
> $ git tag -l "v1.8.5*"
> v1.8.5
> v1.8.5-rc0
> v1.8.5-rc1
> v1.8.5-rc2
> v1.8.5-rc3
> v1.8.5.1
> v1.8.5.2
> v1.8.5.3
> v1.8.5.4
> v1.8.5.5
> 
> If, however, you’re supplying a wildcard pattern to match tag names, the use of -l or --list is mandatory.

Reply:
But the current script you have written is limited, correct?